packages/wrangler/src/api/upgrade.ts

```diff
@@ -0,0 +1,94 @@
+import { writeFileSync, existsSync } from "fs";
+import path from "path";
+import { execaSync } from "execa";
+import { fetch } from "undici";
+import { findWranglerToml } from "../config";
+import { confirm } from "../dialogs";
+import { printWranglerBanner } from "../index";
+import { logger } from "../logger";
+import { readFileSync } from "../parse";
+
+import type { PackageJSON } from "../parse";
+
+export async function upgradeWrangler({
+	yesFlag = false,
```

Comment:
Maybe something more descriptive like `force`, or `skipConfirmation`?

Reply:
Hmmm... `force` would probably be better here, thanks for the suggestion.